.clang-format

```diff
@@ -0,0 +1,46 @@
+# Use this file to configure clang-format
+
```

Comment:
Did you run clang-format on this code ? It doesn't seem to be applied. There are formatting issues in this code.

Reply:
Done